Build failure: digikam

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

1. Enable nixpkgs.cudaSupport
2. Attempt to build digikam

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/303063268

### Relevant log output

```console
Using udevCheckHook
Running phase: qtPreHook
@nix { "action": "setPhase", "phase": "qtPreHook" }
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/xibz7zwpx9isxsklb6i1nagjj7c94m80-source
source root is source
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
applying patch /nix/store/5x7qirfsfs9jpibrj97wd3z6dkk4vs73-disable-tests-download.patch
patching file CMakeLists.txt
applying patch /nix/store/7w9zamcyn5m7azzhwdil691yysl2z92v-325b19fc7f0d04cdc1308f235c207c1ab43e945d.patch
patching file core/libs/widgets/text/localizeselector.cpp
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
fixing cmake files...
cmake flags: -GNinja -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/doc/digikam -DCMAKE_INSTALL_INFODIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/include -DCMAKE_INSTALL_SBINDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/7nlf5v84s4p2yhx327j8495yik60qnzh-gcc-wrapper-14.3.0/bin/strip -DCMAKE_RANLIB=/nix/store/7nlf5v84s4p2yhx327j8495yik60qnzh-gcc-wrapper-14.3.0/bin/ranlib -DCMAKE_AR=/nix/store/7nlf5v84s4p2yhx327j8495yik60qnzh-gcc-wrapper-14.3.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0 -DBUILD_WITH_QT6:BOOL=TRUE -DENABLE_KFILEMETADATASUPPORT:BOOL=TRUE -DENABLE_MEDIAPLAYER:BOOL=TRUE -DENABLE_APPSTYLES:BOOL=TRUE -DCUDA_TOOLKIT_ROOT_DIR=/nix/store/abkkrd3spmzrg8adf64apcyyajpdhhgq-cuda-merged-12.8 -DKDE_INSTALL_EXECROOTDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0 -DKDE_INSTALL_BINDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/bin -DKDE_INSTALL_SBINDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/sbin -DKDE_INSTALL_LIBDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/lib -DKDE_INSTALL_LIBEXECDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/libexec -DKDE_INSTALL_CMAKEPACKAGEDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/lib/cmake -DKDE_INSTALL_QTPLUGINDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/lib/qt-6/plugins -DKDE_INSTALL_PLUGINDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/lib/qt-6/plugins -DKDE_INSTALL_QMLDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/lib/qt-6/qml -DKDE_INSTALL_INCLUDEDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/include -DKDE_INSTALL_LOCALSTATEDIR=/var -DKDE_INSTALL_SHAREDSTATEDIR=/com -DKDE_INSTALL_DATAROOTDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share -DKDE_INSTALL_DATADIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share -DKDE_INSTALL_DOCBUNDLEDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/doc/HTML -DKDE_INSTALL_KCFGDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/config.kcfg -DKDE_INSTALL_KCONFUPDATEDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/kconf_update -DKDE_INSTALL_KAPPTEMPLATESDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/kdevappwizard/templates -DKDE_INSTALL_KFILETEMPLATESDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/kdevfiletemplates/templates -DKDE_INSTALL_KXMLGUIDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/kxmlgui5 -DKDE_INSTALL_KNOTIFYRCDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/knotifications6 -DKDE_INSTALL_ICONDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/icons -DKDE_INSTALL_LOCALEDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/locale -DKDE_INSTALL_SOUNDDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/sounds -DKDE_INSTALL_TEMPLATEDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/templates -DKDE_INSTALL_WALLPAPERDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/wallpapers -DKDE_INSTALL_APPDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/applications -DKDE_INSTALL_DESKTOPDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/desktop-directories -DKDE_INSTALL_MIMEDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/mime/packages -DKDE_INSTALL_METAINFODIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/metainfo -DKDE_INSTALL_QTQCHDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/doc/qch -DKDE_INSTALL_QCHDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/doc/qch -DKDE_INSTALL_MANDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/man -DKDE_INSTALL_INFODIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/info -DKDE_INSTALL_DBUSDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/dbus-1 -DKDE_INSTALL_DBUSINTERFACEDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/dbus-1/interfaces -DKDE_INSTALL_DBUSSERVICEDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/dbus-1/services -DKDE_INSTALL_DBUSSYSTEMSERVICEDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/dbus-1/system-services -DKDE_INSTALL_SYSCONFDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/etc -DKDE_INSTALL_CONFDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/etc/xdg -DKDE_INSTALL_AUTOSTARTDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/etc/xdg/autostart -DKDE_INSTALL_LOGGINGCATEGORIESDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/qlogging-categories6 -DKDE_INSTALL_SYSTEMDUNITDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/lib/systemd -DKDE_INSTALL_SYSTEMDUSERUNITDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/systemd/user -DKDE_INSTALL_EXECROOTDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0 -DKDE_INSTALL_BINDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/bin -DKDE_INSTALL_SBINDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/sbin -DKDE_INSTALL_LIBDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/lib -DKDE_INSTALL_LIBEXECDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/libexec -DKDE_INSTALL_CMAKEPACKAGEDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/lib/cmake -DKDE_INSTALL_QTPLUGINDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/lib/qt-6/plugins -DKDE_INSTALL_PLUGINDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/lib/qt-6/plugins -DKDE_INSTALL_QMLDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/lib/qt-6/qml -DKDE_INSTALL_INCLUDEDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/include -DKDE_INSTALL_LOCALSTATEDIR=/var -DKDE_INSTALL_SHAREDSTATEDIR=/com -DKDE_INSTALL_DATAROOTDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share -DKDE_INSTALL_DATADIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share -DKDE_INSTALL_DOCBUNDLEDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/doc/HTML -DKDE_INSTALL_KCFGDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/config.kcfg -DKDE_INSTALL_KCONFUPDATEDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/kconf_update -DKDE_INSTALL_KAPPTEMPLATESDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/kdevappwizard/templates -DKDE_INSTALL_KFILETEMPLATESDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/kdevfiletemplates/templates -DKDE_INSTALL_KXMLGUIDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/kxmlgui5 -DKDE_INSTALL_KNOTIFYRCDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/knotifications6 -DKDE_INSTALL_ICONDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/icons -DKDE_INSTALL_LOCALEDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/locale -DKDE_INSTALL_SOUNDDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/sounds -DKDE_INSTALL_TEMPLATEDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/templates -DKDE_INSTALL_WALLPAPERDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/wallpapers -DKDE_INSTALL_APPDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/applications -DKDE_INSTALL_DESKTOPDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/desktop-directories -DKDE_INSTALL_MIMEDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/mime/packages -DKDE_INSTALL_METAINFODIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/metainfo -DKDE_INSTALL_QTQCHDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/doc/qch -DKDE_INSTALL_QCHDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/doc/qch -DKDE_INSTALL_MANDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/man -DKDE_INSTALL_INFODIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/info -DKDE_INSTALL_DBUSDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/dbus-1 -DKDE_INSTALL_DBUSINTERFACEDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/dbus-1/interfaces -DKDE_INSTALL_DBUSSERVICEDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/dbus-1/services -DKDE_INSTALL_DBUSSYSTEMSERVICEDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/dbus-1/system-services -DKDE_INSTALL_SYSCONFDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/etc -DKDE_INSTALL_CONFDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/etc/xdg -DKDE_INSTALL_AUTOSTARTDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/etc/xdg/autostart -DKDE_INSTALL_LOGGINGCATEGORIESDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/qlogging-categories6 -DKDE_INSTALL_SYSTEMDUNITDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/lib/systemd -DKDE_INSTALL_SYSTEMDUSERUNITDIR=/nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0/share/systemd/user -DECM_DIR=/nix/store/rlq1vkgqc9avngp84nkih181hrmcjlia-extra-cmake-modules-5.116.0/share/ECM/cmake -DPROTOC_EXE=/nix/store/zrkxcyb50mh9vcq31s56cvg3zfl2pn01-protobuf-31.1/bin/protoc -DProtobuf_PROTOC_EXE=/nix/store/zrkxcyb50mh9vcq31s56cvg3zfl2pn01-protobuf-31.1/bin/protoc -DProtobuf_PROTOC_EXECUTABLE=/nix/store/zrkxcyb50mh9vcq31s56cvg3zfl2pn01-protobuf-31.1/bin/protoc
-- CMake version: 3.31.7
-- The CXX compiler identification is GNU 14.3.0
-- The C compiler identification is GNU 14.3.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/7nlf5v84s4p2yhx327j8495yik60qnzh-gcc-wrapper-14.3.0/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/7nlf5v84s4p2yhx327j8495yik60qnzh-gcc-wrapper-14.3.0/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- ----------------------------------------------------------------------------------
-- Starting CMake configuration for digikam
-- 
-- Target Build Type: Release
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Performing Test HAVE_STDATOMIC
-- Performing Test HAVE_STDATOMIC - Success
-- Found WrapAtomic: TRUE
-- Suitable Qt6 >= 6.4.0 detected: '6.9.1'.
[0mInstalling in /nix/store/dglq8bik93gxnk6dsvavg2s7cq66mj7p-digikam-8.6.0. Run /build/source/build/prefix.sh to set the environment for digikam.[0m
-- Looking for __GLIBC__
-- Looking for __GLIBC__ - found
-- Performing Test _OFFT_IS_64BIT
-- Performing Test _OFFT_IS_64BIT - Success
-- Performing Test HAVE_DATE_TIME
-- Performing Test HAVE_DATE_TIME - Success
-- ----------------------------------------------------------------------------------
-- Top level compilation configuration for digikam
-- 
-- digikam will be compiled....................... YES
-- Translations will be compiled.................. YES
-- Tests code will be compiled.................... NO
-- ----------------------------------------------------------------------------------
-- Check dependencies for digikam version 8.6.0
-- 
-- Found OpenGL: /nix/store/iyy1g70fhkz3hsrckbmbqgxik1j9779c-libglvnd-1.7.0/lib/libOpenGL.so
-- Found WrapOpenGL: TRUE
-- Found WrapVulkanHeaders: /nix/store/sgv2i8cvlpi8ly21xwdl8dvgia50x2mc-vulkan-headers-1.4.313.0/include
-- Found Cups: /nix/store/wj5sc0i81fb6hcz802gmsgdsjll79wfc-cups-2.4.12-lib/lib/libcups.so (found version "2.4.12")
-- Found KF6XmlGui: /nix/store/sd2ckbcfvd1x1n68lgxjhsx3px6i9dic-kxmlgui-6.16.0-dev/lib/cmake/KF6XmlGui/KF6XmlGuiConfig.cmake (found version "6.16.0")
-- Found KF6CoreAddons: /nix/store/1ffccdygfl56zm02prqnr2n0j5v6p6jn-kcoreaddons-6.16.0-dev/lib/cmake/KF6CoreAddons/KF6CoreAddonsConfig.cmake (found version "6.16.0")
-- Found KF6Config: /nix/store/nlayykj1a0sk5lzjwg63xc1k1ax3q0dj-kconfig-6.16.0-dev/lib/cmake/KF6Config/KF6ConfigConfig.cmake (found version "6.16.0")
-- Found KF6Service: /nix/store/iklj1banc0ggmr0s2w5ll4fzp01v2vvp-kservice-6.16.0-dev/lib/cmake/KF6Service/KF6ServiceConfig.cmake (found version "6.16.0")
-- Found X11: /nix/store/l1qkka1svxk8s0myynmb6kgl0ni19mjk-xorgproto-2024.1/include
-- Looking for XOpenDisplay in /nix/store/pahwl2rq51dmwrn8czks27yy3sa3byg9-libX11-1.8.12/lib/libX11.so;/nix/store/844cgxkyzi1nrilvamxr08gs9l278gx9-libXext-1.3.6/lib/libXext.so
-- Looking for XOpenDisplay in /nix/store/pahwl2rq51dmwrn8czks27yy3sa3byg9-libX11-1.8.12/lib/libX11.so;/nix/store/844cgxkyzi1nrilvamxr08gs9l278gx9-libXext-1.3.6/lib/libXext.so - found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - found
-- Looking for IceConnectionNumber in ICE
-- Looking for IceConnectionNumber in ICE - found
-- Found KF6WindowSystem: /nix/store/hld0rv33ykk0d1ap4l0cbmx6rnnxhgfr-kwindowsystem-6.16.0-dev/lib/cmake/KF6WindowSystem/KF6WindowSystemConfig.cmake (found version "6.16.0")
-- Found KF6Solid: /nix/store/1zi8ks1v5l8g7paq92g8icbplfrk0pc0-solid-6.16.0-dev/lib/cmake/KF6Solid/KF6SolidConfig.cmake (found version "6.16.0")
-- Found Gettext: /nix/store/idymwwnvsx6si5ihybrl1zkpgp38y617-gettext-0.25/bin/msgmerge (found version "0.25")
-- Found KF6I18n: /nix/store/7vilraf6kd3qjksa3jfw0xxh6333vzzk-ki18n-6.16.0-dev/lib/cmake/KF6I18n/KF6I18nConfig.cmake (found version "6.16.0")
-- Found KF6: success (found suitable version "6.16.0", minimum required is "5.95.0") found components: XmlGui CoreAddons Config Service WindowSystem Solid I18n
-- Found KF6KIO: /nix/store/xw5bl7lf53klw1jsxd56sdjlkdz2mpv0-kio-6.16.0-dev/lib/cmake/KF6KIO/KF6KIOConfig.cmake (found version "6.16.0")
-- Found KF6IconThemes: /nix/store/r27w4pd0bkac3jfih41jnpahf7bgjdrc-kiconthemes-6.16.0-dev/lib/cmake/KF6IconThemes/KF6IconThemesConfig.cmake (found version "6.16.0")
-- Found KF6ThreadWeaver: /nix/store/phb3jfj9dc0ld26yawsfncfrf4sihzmd-threadweaver-6.16.0-dev/lib/cmake/KF6ThreadWeaver/KF6ThreadWeaverConfig.cmake (found version "6.16.0")
-- Found KF6NotifyConfig: /nix/store/hdf6mbj00wf3yp6wjw6nf6hscdmi4jd1-knotifyconfig-6.16.0-dev/lib/cmake/KF6NotifyConfig/KF6NotifyConfigConfig.cmake (found version "6.16.0")
-- Found KF6Notifications: /nix/store/7slrf5q17vbaiym41gaknkpbgkvc44pl-knotifications-6.16.0-dev/lib/cmake/KF6Notifications/KF6NotificationsConfig.cmake (found version "6.16.0")
-- Found KF6Sonnet: /nix/store/48kabf3lq23zv2ss837dzn5sv19v8xln-sonnet-6.16.0-dev/lib/cmake/KF6Sonnet/KF6SonnetConfig.cmake (found version "6.16.0")
-- Found KF6FileMetaData: /nix/store/1rnhhwba8164bdxiiwarm0ac5da3nqav-kfilemetadata-6.16.0-dev/lib/cmake/KF6FileMetaData/KF6FileMetaDataConfig.cmake (found version "6.16.0")
[33mCMake Warning at core/cmake/rules/RulesKDEFramework.cmake:70 (find_package):
  By not providing "FindKSaneWidgets6.cmake" in CMAKE_MODULE_PATH this
  project has asked CMake to find a package configuration file provided by
  "KSaneWidgets6", but CMake did not find one.

  Could not find a package configuration file provided by "KSaneWidgets6"
  with any of the following names:

    KSaneWidgets6Config.cmake
    ksanewidgets6-config.cmake

  Add the installation prefix of "KSaneWidgets6" to CMAKE_PREFIX_PATH or set
  "KSaneWidgets6_DIR" to a directory containing one of the above files.  If
  "KSaneWidgets6" provides a separate development package or SDK, be sure it
  has been installed.
Call Stack (most recent call first):
  core/CMakeLists.txt:41 (include)

[0m
-- Found KF6CalendarCore: /nix/store/swkw75qzlyvsr8pdz0ddnww2qwvcfb04-kcalendarcore-6.16.0-dev/lib/cmake/KF6CalendarCore/KF6CalendarCoreConfig.cmake (found version "6.16.0")
-- KF6::KIOWidgets include dirs: /nix/store/xw5bl7lf53klw1jsxd56sdjlkdz2mpv0-kio-6.16.0-dev/include/KF6/KIOWidgets
-- Found JPEG: /nix/store/15rkcy8rnhnv4cbai5q7dgsmhpj7nznj-libjpeg-turbo-3.1.1/lib/libjpeg.so (found version "62")
-- Libjpeg version: 62
-- Using libjpeg sub-directory: libjpeg/62
-- Looking for LibJpeg Version...
[0mERRORThe Libjpeg API is less than 8.0. At least libjpeg >= 8 is required to support DNG compression.[0m
-- Found TIFF: /nix/store/2vr72axg8pklihpzzg0mgwlzidsvvd63-libtiff-4.7.0-dev/lib/cmake/tiff (found version "4.7.0")
-- Found ZLIB: /nix/store/abch1r0gnbpikbp9n4x6mm8dwqwfrib6-zlib-1.3.1/lib/libz.so (found version "1.3.1")
-- Found PNG: /nix/store/4h3bvqsxa229qdqrw7z9spdzsqahs2hn-libpng-apng-1.6.47/lib/libpng.so (found version "1.6.47")
[33mCMake Warning (dev) at core/CMakeLists.txt:52 (find_package):
  Policy CMP0167 is not set: The FindBoost module is removed.  Run "cmake
  --help-policy CMP0167" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

This warning is for project developers.  Use -Wno-dev to suppress it.
[0m
-- Found Boost: /nix/store/ir0758kmac2mk5852s7cp65ss8ins92p-boost-1.87.0-dev/lib/cmake/Boost-1.87.0/BoostConfig.cmake (found version "1.87.0")
-- Found PkgConfig: /nix/store/rxz7i1sj19nrcmyk4sg49i2h1vfaazby-pkg-config-wrapper-0.29.2/bin/pkg-config (found version "0.29.2")
-- Checking for module 'lcms2'
--   Found lcms2, version 2.17
-- Found lcms version 2.17, /nix/store/r35f0h5lnips6hwv19kv0avdlgvl999g-lcms2-2.17/lib/liblcms2.so
-- Found EXPAT: /nix/store/4ciq67pdwyc43a5y1h63wcipl54f938w-expat-2.7.1/lib/libexpat.so (found version "2.7.1")
Package libde265 was not found in the pkg-config search path.
Perhaps you should add the directory containing `libde265.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libde265', required by 'libheif', not found
Package libde265 was not found in the pkg-config search path.
Perhaps you should add the directory containing `libde265.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libde265', required by 'libheif', not found
Package libde265 was not found in the pkg-config search path.
Perhaps you should add the directory containing `libde265.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libde265', required by 'libheif', not found
Package libde265 was not found in the pkg-config search path.
Perhaps you should add the directory containing `libde265.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libde265', required by 'libheif', not found
-- Found Libheif: /nix/store/qf5vzhcvzbxjwkh661d7hpkcwfgdf7jm-libheif-1.19.8-dev/include
-- HEIF_FOUND        = TRUE
-- HEIF_INCLUDE_DIRS = /nix/store/qf5vzhcvzbxjwkh661d7hpkcwfgdf7jm-libheif-1.19.8-dev/include
-- HEIF_LIBRARIES    = /nix/store/8ivqhpgll99gsmvw0gnc4r3s54cl3jxb-libheif-1.19.8/lib/libheif.so
-- HEIF_DEFINITIONS  = 
-- HEIF_VERSION      = 1.19.8
-- Found Libjxl: /nix/store/q9xq58ifiq0l7blx6i12d0vi2f87ilks-libjxl-0.11.1-dev/include
-- JXL_FOUND        = TRUE
-- JXL_INCLUDE_DIRS = /nix/store/q9xq58ifiq0l7blx6i12d0vi2f87ilks-libjxl-0.11.1-dev/include
-- JXL_LIBRARIES    = /nix/store/y92is85g9d011i7fxm5nz3j433s30krd-libjxl-0.11.1/lib/libjxl.so;/nix/store/y92is85g9d011i7fxm5nz3j433s30krd-libjxl-0.11.1/lib/libjxl_threads.so
-- JXL_DEFINITIONS  = 
-- JXL_VERSION      = 0.11.1
-- Checking for module 'x265'
--   Found x265, version 4.1
-- Found X265: /nix/store/h585qivhp55nki0x8h1nv5d3prdcnrjd-x265-4.1-dev/include
-- X265_FOUND        = TRUE
-- X265_INCLUDE_DIRS = /nix/store/h585qivhp55nki0x8h1nv5d3prdcnrjd-x265-4.1-dev/include
-- X265_LIBRARIES    = /nix/store/kc17grcx04y0395mcqlwylmnxjwq8758-x265-4.1/lib/libx265.so
-- X265_DEFINITIONS  = 
-- Found LibExiv2: /nix/store/sybk0jvmh7ybingnli440rjkawvjfy5a-exiv2-0.28.5-lib/lib/libexiv2.so (found version "0.28.5")
[31mCMake Error at /nix/store/hvn9cprr3p61gjlahk9pq4bmp3zaxj4a-cmake-3.31.7/share/cmake-3.31/Modules/FindCUDAToolkit.cmake:896 (message):
  Could not find nvcc, please set CUDAToolkit_ROOT.
Call Stack (most recent call first):
  /nix/store/vnfyqc6yvaz6qbws68n61awbxppbyv34-opencv-4.11.0/lib/cmake/opencv4/OpenCVConfig.cmake:86 (find_package)
  /nix/store/vnfyqc6yvaz6qbws68n61awbxppbyv34-opencv-4.11.0/lib/cmake/opencv4/OpenCVConfig.cmake:108 (find_host_package)
  core/cmake/macros/MacroOpenCV.cmake:27 (find_package)
  core/cmake/rules/RulesLibOpenCV.cmake:7 (DETECT_OPENCV)
  core/CMakeLists.txt:77 (include)

[0m
-- Configuring incomplete, errors occurred!
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.15.7, NixOS, 25.11 (Xantusia), 25.11.20250719.2df1483`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.4`
 - nixpkgs: `/nix/store/pp4mcsvkidl3wqcx09zq422ka5mghblz-source`

### Notify maintainers



---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/di/digikam/package.nix
+++ b/pkgs/by-name/di/digikam/package.nix
@@ -4,7 +4,6 @@
   lib,
   fetchFromGitLab,
   fetchgit,
-  fetchpatch,
 
   cmake,
   ninja,
@@ -63,25 +62,18 @@ in
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "digikam";
-  version = "8.6.0";
+  version = "8.7.0";
 
   src = fetchFromGitLab {
     domain = "invent.kde.org";
     owner = "graphics";
     repo = "digikam";
-    rev = "v${finalAttrs.version}";
-    hash = "sha256-CMyvNOAlIqD6OeqUquQ+/sOiBXmEowZe3/qmaXxh0X0=";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-9t6tXrege3A5x5caUEfho23Pin7dON+e6x94rXC8XYE=";
   };
 
   patches = [
     ./disable-tests-download.patch
-
-    # Fix build with Qt 6.9
-    # FIXME: remove in next update
-    (fetchpatch {
-      url = "https://invent.kde.org/graphics/digikam/-/commit/325b19fc7f0d04cdc1308f235c207c1ab43e945d.patch";
-      hash = "sha256-bsxaNuLuU9MyDRmenOqO4JuzsbpUvfKQwcSCDfLHoWQ=";
-    })
   ];
 
   strictDeps = true;
@@ -104,7 +96,7 @@ stdenv.mkDerivation (finalAttrs: {
   # build inputs.
 
   buildInputs = [
-    opencv
+    opencv.cxxdev
     libtiff
     libpng
     # TODO: Figure out how on earth to get it to pick up libjpeg8 for

